### PR TITLE
fix(ci): remove override file

### PR DESCRIPTION
**Pull request by fzipi**
Signed-off-by: Felipe Zipitria <felipe.zipitria@owasp.org>

Same as #33 , trying to debug nginx build.

### CODE REVIEW

**Review by franbuehler — APPROVED** [review]
Visual check looks good but yes, failing build.

**Comment by fzipi** [comment]
Closing in favor of #32 from new contributor!

**Comment by fzipi** [comment]
@franbuehler Can you review this one again? With the latest changes to the modsecurity-docker, we needed to move this here after removing the override file.

File: README.md [hunk 1]
```diff
@@ -89,6 +94,8 @@ The following environment variables are available to configure the CRS container
 | MODSEC_RESP_BODY_LIMIT | ModSecurity integer value indicating the maximum response body size  accepted for buffering (Default: 524288) |
 | MODSEC_PCRE_MATCH_LIMIT | ModSecurity integer value indicating the limit for the number of internal executions in the PCRE function (Default: 1000) |
 | MODSEC_PCRE_MATCH_LIMIT_RECURSION | ModSecurity integer value indicating the limit for the depth of recursion when calling PCRE function (Default: 1000) |
+| MODSEC_DEFAULT_PHASE1_ACTION | ModSecurity string with the contents for the default action in phase 1 (Default: `'phase:1,log,auditlog,pass,tag:\'\${MODSEC_TAG}\''`) |
```
[the remark this answers is not in the document]
Comment (fzipi) [reply README.md:97]: Good catch. They were moved, that's why they were not here.

**Comment by fzipi** [comment]
After removing also vars in base container in https://github.com/coreruleset/modsecurity-docker/pull/77, this one is ready to be merged.

**Pull request merged**